crates/nu-cli/src/repl.rs

```diff
@@ -238,6 +238,24 @@ pub fn evaluate_repl(
             use_color,
         );
 
+        start_time = std::time::Instant::now();
+        let config = &engine_state.get_config().clone();
```

Comment:
Why do we need the config clone here? It doesn't seem necessary since you clone the pre_prompt hook anyway and another config is defined below: https://github.com/nushell/nushell/pull/10346/files#diff-665392adc0e17b5c199c59ae562f98f0673e18effdb813dc3578a1e55e191e42R260